client/public/js/components/ObservationCreate.js

```diff
@@ -0,0 +1,72 @@
+var React = require('react'),
+    Navigation = require('react-router').Navigation,
+    EventDetailStore = require('../stores/EventDetailStore'),
+    EventDetailActions = require('../actions/EventDetailActions');
+
+var ObservationCreate = React.createClass({
+  mixins: [Navigation],
+
+  //initially sets the text of the observation form to blank
+  //will be filed in the form
+  getInitialState: function() {
+    return {
+      name: EventDetailStore.getCurrentUser(),

```

Comment:
Perhaps `getCurrentUser` should be a method of `AppStore` since I could see it being used in other components outside of events. Thoughts?